curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/store-multiple-values-in-one-variable-using-javascript-arrays.md

```diff
@@ -13,7 +13,7 @@ With JavaScript `array` variables, we can store several pieces of data in one pl
 
 You start an array declaration with an opening square bracket, end it with a closing square bracket, and put a comma between each entry, like this:
 
-`var sandwich = ["peanut butter", "jelly", "bread"]`.
+`var sandwich = ["peanut butter", "jelly", "bread"]`
```

Comment:
Off-topic: Should this not be within triple backticks with a code language?

Reply:
It could be, but as long as it only contains a single back tick code block, it will work.